CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+1.3.3 --- 2020-10-05
+____________________
+
+* Adding option to include a body parameter in requests.
```

Comment:
Since we are extending the interface of the library, shouldn't this have been version 1.4?

Reply:
@nedbat You're right, I can update it - I haven't merged my PR for Registrar yet, so it shouldn't be an issue.